client/src/Components/WorkCard/WorkCard.tsx

```diff
@@ -1,7 +1,8 @@
-import React from 'react';
-import { Card } from 'antd';
+import React, { useState } from 'react';
+import { Card, Modal, message } from 'antd';
+import { EditOutlined, DeleteOutlined } from '@ant-design/icons';
+import { useTranslation } from 'react-i18next';
 import './style.css';
-import { EditOutlined, SettingOutlined } from '@ant-design/icons';
 
 const { Meta } = Card;
 interface WorkCardProps {
```

Comment:
can you move this interface to utils/types/index.ts ?

Reply:
done